source/includes/plugin_info/_050-show.md.erb

```diff
@@ -18,55 +18,60 @@ ETag: "4167e3ec81fdac0fb29d854b36ceb981"
 {
   "_links": {
     "self": {
-      "href": "https://ci.example.com/go/api/admin/plugin_info/my_plugin"
+      "href": "ci.example.com/go/api/admin/plugin_info/sns.notification.plugin"
```

Comment:
Can you add https:// back and replace sns.notification.plugin with something more generic like my_plugin